models/models.go

```diff
@@ -28,7 +28,7 @@ const (
 	Partial  PayloadStatus = "partial"
 	Pending  PayloadStatus = "pending"
 	Running  PayloadStatus = "running"
-	Complete PayloadStatus = "complete"
+	Complete PayloadStatus = "completed_at"
```

Comment:
This references the status of export being `complete`, which is different from the time it was completed `completed_at`.